src/components/popover-container/popover-container.component.tsx

```diff
@@ -44,20 +45,24 @@ export const renderOpen = ({
   id,
   "aria-expanded": ariaExpanded,
   "aria-haspopup": ariaHasPopup,
-}: RenderOpenProps) => (
-  <PopoverContainerOpenIcon
-    tabIndex={tabIndex}
-    onClick={onClick}
-    data-element={dataElement}
-    ref={ref}
-    aria-label={ariaLabel}
-    aria-haspopup={ariaHasPopup}
-    aria-expanded={ariaExpanded}
-    id={id}
-  >
-    <Icon type="settings" />
-  </PopoverContainerOpenIcon>
-);
+  ...rest
+}: RenderOpenProps) => {
+  return (
+    <PopoverContainerOpenIcon
+      tabIndex={tabIndex}
+      onClick={onClick}
+      data-element={dataElement}
+      ref={ref}
+      aria-label={ariaLabel}
+      aria-haspopup={ariaHasPopup}
+      aria-expanded={ariaExpanded}
+      id={id}
+      {...rest}
```

Comment:
**suggestion**: what do you think about replacing spreading the remaining props with passing `isOpen` directly?

```suggestion
      isOpen={isOpen}
```

With the open icon being a styled `button`, spreading `rest` would technically allow consumers to pass any valid html attributes to it.

Reply:
I don't think they can here as it's just the default that renders if they don't set the prop etc. However, I agree it's probably better to explicitly destructure here